lib/internal/Magento/Framework/App/Bootstrap.php

```diff
@@ -108,6 +108,13 @@ class Bootstrap
      */
     private $factory;
 
+    /**
+     * Application Object
+     *
+     * @var ObjectManagerFactory
```

Comment:
Should this be `AppInterface` ?
I would also recommend to change from `$app` to `$application`.